src/core/utils.js

```diff
@@ -680,10 +680,10 @@ export function addId(elem, pfx = "", txt = "", noLC = false) {
 export function getTextNodes(el, exclusions = [], options = { wsNodes: true }) {
   const exclusionQuery = exclusions.join(", ");
   const acceptNode = (/** @type {Text} */ node) => {
-    if (node.parentElement.closest(exclusionQuery)) {
+    if (!options.wsNodes && /^\s*$/.test(node.data)) {
```

Comment:
From a regex objector: How about `!node.data.trim()`?

Reply:
yeah, that's clearer.  However, semantically, I kinda liked the regex. I'll change it tho.